features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumApplicationScope.kt

```diff
@@ -155,6 +169,34 @@ internal class RumApplicationScope(
         }
     }
 
+    @WorkerThread
+    private fun sendApplicationStartEvent(eventTime: Time, writer: DataWriter<Any>) {
+        val processImportance = DdRumContentProvider.processImportance
+        val isForegroundProcess = processImportance ==
+            ActivityManager.RunningAppProcessInfo.IMPORTANCE_FOREGROUND
+        if (isForegroundProcess) {
+            val processStartTime = appStartTimeProvider.appStartTimeNs
```

Comment:
minor: since the processStartTime is in ns, should we not rename to processStartTimeNs?